bundles/CoreBundle/Resources/config/templating.yaml

```diff
@@ -52,7 +52,7 @@ services:
     # GLOSSARY
 
     Pimcore\Tool\Glossary\Processor:
-        public: false
+        public: true #For Tests
```

Comment:
Not really cool, to make it `public` only for tests, no? Doesn't Codeception has a better way for this? Like the Symfony test container?

Reply:
no idea

Reply:
maybe this helps https://pimcore.com/docs/pimcore/current/Development_Documentation/Development_Tools_and_Details/Testing/Testing_Services.html#page_Testing-Symfony-Services

Reply:
done